mta-sts.md

```diff
@@ -302,7 +302,7 @@ sender fetches a HTTPS resource (policy) from a host at `policy.mta-sts` in the
 Policy Domain. The policy is served from a "well known" URI:
 `https://policy.mta-sts.example.com/.well-known/mta-sts/current`. To consider 
```

Comment:
aah, i would suggest to mention RFC5785 as reference (ideally closer to when we mention .well-known for the first time.